less/process.less

```diff
@@ -0,0 +1,301 @@
+@process-xl-radius: 64px;
+@process-xl-stroke: 3px;
+@process-xl-circumference: (pi() * (2 * (@process-xl-radius - 1)));
+@process-xl-icon-size: 64px;
+@process-xl-icon-stroke: 8px;
+/*
+// Keep these sizes in case we want to add these back in.
+@process-lg-radius: 48px;
+@process-lg-stroke: 2.7px;
+@process-lg-circumference: (pi() * (2 * (@process-lg-radius - 1)));
+@process-lg-icon-size: 48px;
+@process-lg-icon-stroke: 6px;
+
+@process-md-radius: 36px;
+@process-md-circumference: (pi() * (2 * (@process-md-radius - 1)));
+@process-md-stroke: 2.5px;
+@process-md-icon-size: 36px;
+@process-md-icon-stroke: 4px;
+*/
```

Comment:
I originally made variants at 5 different sizes, we don't currently have use cases for the others, so I removed them from the code, but I left the variables as tuning the stroke widths was a little tricky, and they may well come back.